recipes/streamz/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set org = "mrocklin" %}
+{% set name = "streamz" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "c8621a9552225c5bb8ece69618a35a0fd262087040043f0c9b8ef7ac41e40f17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/{{ org }}/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - zict
+    - tornado
+    - toolz
+    - setuptools
+  run:
+    - python
+    - zict
+    - tornado
+    - toolz
+
+test:
+  imports:
+    - streamz
+    - streamz.core
+
+about:
+  home: https://github.com/{{ org }} /{{ name }}/
```

Comment:
Can you just write out the explicit url here and in the `doc_url` and `dev_url`? It might be nitpicky on my part, but I like being able to look at a recipe and then cut-and-paste urls without dealing with the jinja templating stuff.